src/containers/rosterChecker/rosterChecker.jsx

```diff
@@ -0,0 +1,258 @@
+/**
+ * Roster Checker is a tool to compare the arrangement against a roster and see who is missing 
+ * from the roster.
+ */
+
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { HotTable } from '@handsontable/react';
+import { Card, CardHeader, CardContent, TextField, Grid, Typography } from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
+
+import cloneDeep from 'lodash/cloneDeep';
+import isEmpty from 'lodash/isEmpty';
+import { withSnackbar } from 'notistack';
+
+import { setRealData } from 'actions/real/real'
+import { getArrangement } from 'services/arrangementService'
+
+const NAME_FIELD = "name";
+const VALIDITY_FIELD = "validity";
+const PRESENT = "present";
+const MISSING = "missing";
+const SHAMROCK_GREEN = "#0F9D58";
+const CINNABAR_RED = "#EA4335";
+
+const styles = theme => ({
+    sheet: {
+        marginLeft: 10,
+        marginTop: 10,
+        marginBottom: 10,
+        marginRight: 10,
+        borderStyle: "solid",
+        borderWidth: "1px",
+        borderColor: "#777"
+    },
+    card: {
+        background:"#fafafa"
+    },
+    cardHeader: {
+        paddingLeft: 10,
+        paddingTop: 10,
+        paddingBottom: 0,
+        paddingRight: 10
+    },
+    cardContent: {
+        height: "calc(100vh - 250px)",
+        overflow: "scroll"
+    }
+})
+
+class SheetView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            arrangementId: "",
+            currentRoster: [],
+            newRoster: [],
+        }
+    }
+
+    generateColumnHeaders = () => {
+        return this.generateColumnDefs().map(col => col.data);
+    }
+
+    generateColumnDefs = () => {
+        return [
+            {
+                data: NAME_FIELD,
+                width: 220
+            },
+            {  
+                data: VALIDITY_FIELD,
+                width: 220,
+                readOnly: true,
+                renderer: this.renderValidity
+            }
+        ];
+    }
+
+    generateCurrentRoster = () => {
+        const itemList = []
+        console.log(this.props.real)
+        if (!isEmpty(this.props.real)) {
```

Comment:
why is this called real? what is 'real'?

Reply:
jeff chiu named it as such